src/test/java/unit/Group.java

```diff
@@ -96,16 +98,45 @@ void listDescribeUsersReflectCreateDelete() throws IOException {
         1, groupDescribed.numMembers, "group # members matches describe output after create");
 
     // `terra group delete --name=$name`
-    TestCommand.runCommandExpectSuccess("group", "delete", "--name=" + name, "--quiet");
+    TestCommand.runCommandExpectSuccess("group", "delete", "--name=" + name1, "--quiet");
 
     // `terra group list`
     groupList =
         TestCommand.runAndParseCommandExpectSuccess(new TypeReference<>() {}, "group", "list");
 
     // check that the group is not included in the list output
     matchedGroup =
-        groupList.stream().filter(listedGroup -> listedGroup.name.equals(name)).findAny();
+        groupList.stream().filter(listedGroup -> listedGroup.name.equals(name1)).findAny();
     assertTrue(matchedGroup.isEmpty(), "group does not appear in list after delete");
+
+    // create another group
+    String name2 = SamGroups.randomGroupName();
+    TestCommand.runAndParseCommandExpectSuccess(
+        UFGroup.class, "group", "create", "--name=" + name2);
+
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list");
+
+    // use regular expression testing the table format and content inside
+    assertTrue(cmd.stdErr == null || cmd.stdErr.isEmpty());
+    assertEquals(0, cmd.exitCode, "group list returned successfully");
+    String[] rows = cmd.stdOut.split("\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals("NAME", rowHead[0].trim().replace("\r", ""));
```

Comment:
You don't need the `.trim.replace()` for these. The split takes care of the trimming, and there are no carriage returns coming from the table printer (or any other CLI output that I'm aware of).